kentik_api_library/kentik_api/kentik_api.py

```diff
@@ -47,5 +56,13 @@ def __init__(
         self.batch = BatchAPI(connector)
         self.alerting = AlertingAPI(connector)
 
+        self.synthetics = KentikSynthClient(
```

Comment:
Not sure if I understand correctly. In this example above, the SyntheticsAPI would create the connector instance by itself?
If so, this would break the concept of unit testing using stub json data fed into StubAPIConnector that is provided in API constructor...

Reply:
After thinking a bit more, I don't quite see the use case for api connector in case of gRPC, do we want to forge HTTP/2 requests manually?

Reply:
eventually, introduced api connector for synthetics.